.travis.yml

```diff
@@ -16,7 +16,7 @@ cache:
   directories:
     - '$HOME/.sonar/cache'
 
-script: mvn clean install sonar:sonar
+script: mvn clean install sonar:sonar -q -DskipITs=true
```

Comment:
This change - at least the addition of the `-q` flag - needs to be made the [`snapshot-deploy.sh`](https://github.com/connexta/admin-console-beta/blob/master/snapshot-deploy.sh) script as well.